src/components/Nav/Nav.tsx

```diff
@@ -1,5 +1,39 @@
-export const Nav = () => {
-  return <div className="bg-slate-200 px-10 py-5">Nav</div>;
+import categories from "@/data/categories.json";
+import categoryGroups from "@/data/category-groups.json";
+import { CategoryGroup } from "@/types/Category";
+
+export const Subheader = () => {
+  const getSubcategoriesForCategory = (groupId: number) =>
+    categories.filter((subcategory) => subcategory.groupId === groupId);
+
+  return (
+    <nav className="bg-white m-0 p-4 text-center text-sm md:text-base">
+      <ul className="flex justify-center list-none space-x-5 lg:justify-start">
+        {categoryGroups.map((category: CategoryGroup) => (
+          <li className="relative group" key={category.id}>
+            <a className="text-black hover:text-red-500 cursor-pointer py-3">
+              {category.name}
+            </a>
+            <ul
+              className="absolute hidden group-hover:block z-10 bg-white border border-gray-400 rounded mt-2 p-2 lg:w-36 
+              left-1/2 transform -translate-x-1/2 lg:left-auto lg:transform-none"
+            >
+              {getSubcategoriesForCategory(category.id).map((subcategory) => (
+                <li className="mb-2 last:mb-0" key={subcategory.id}>
+                  <a
+                    href={`/category/${category.name}/${subcategory.name}`}
+                    className="block text-black hover:text-red-500"
+                  >
+                    {subcategory.name}
+                  </a>
```

Comment:
En lugar a un anchor (a) utiliza un Link de react router 


```suggestion
                  <a
                    Link to={`/category/${category.name}/${subcategory.name}`}
                    className="block text-black hover:text-red-500"
                  >
                    {subcategory.name}
                  </a>
```

tambien, de ser posible utiliza la librería lodash para usar el kebab case en los nombres de del grupo y subcategoria https://www.geeksforgeeks.org/lodash-_-kebabcase-method/

```suggestion
                  <a
                    Link to={`/category/${_.kebabCase(category.name)}/${_.kebabCase(subcategory.name)}`}
                    className="block text-black hover:text-red-500"
                  >
                    {subcategory.name}
                  </a>
```

Esto es para que queden en minusculas y separadas por guioncitos